src/app/meet/[meetingId]/meeting-components.tsx

```
@@ -0,0 +1,1072 @@
+					roomId,
+					workspaceId,
+					transcriptChunk: saveBufferRef.current.trim(),
+				});
+				saveBufferRef.current = "";
+			} catch (e) {
+				console.error("Failed to save transcript chunk:", e);
+			}
+		}
+	}, [roomId, workspaceId, saveTranscript]);
+
+	const startRecording = useCallback(async () => {
```

Biome (2.4.15)

[error] This hook does not specify its dependency on broadcast.

(lint/correctness/useExhaustiveDependencies)